app/views/admin/dashboard/_recent_projects_grid_definition.html.slim

```diff
@@ -0,0 +1,27 @@
+= define_grid(@recent_projects_grid, hide_csv_button: true,
+    html: {class: 'rowlink table-hover', data: {link: @csv_mode ? nil : 'row'}}) do |g|
+
+    - g.blank_slate do
+      - @empty_list = true
+      - content_tag(:div, t('common.no_records'), class: 'alert alert-info')
+
+    - g.column name: t('common.id'), attribute: 'id', filter: false, in_csv: false,
+      ordering: false do |project|
+      - if project.type == "BasicProject"
+        - link_to(project.id, admin_basic_project_path(project.id))
+      - else
+        - link_to(project.id, admin_loan_path(project.id))
+    - g.column name: csv_id, attribute: 'id', in_html: false, filter: false
```

Comment:
This line may not be needed, given that this table will not be exported to CSV. There are a few spots where in_html and in_csv are defined, but may be not needed.